### DISCUSSION TITLE
blog/react-query-api-design-lessons-learned

### DISCUSSION THREAD

**Discussion by giscus** [TkDodo/blog-comments#111] [Announcements]
# blog/react-query-api-design-lessons-learned

In this talk, React Query maintainer Dominik will walk us through some of the API design choices that were made in React Query to get to that DX. You'll hear stories about things that went well, but also about tradeoffs and mistakes that were made, and what lessons we can all learn from those.

https://tkdodo.eu/blog/react-query-api-design-lessons-learned

**Comment by LovesWorking** [comment]
Great talk! Thanks so much.
